src/java/test/org/apache/zookeeper/server/quorum/CommitProcessorConcurrencyTest.java

```diff
@@ -106,6 +106,7 @@ public void shutdown() {
 
                 @Override
                 public void notifyStopping(String threadName, int errorCode) {
+                    Assert.fail("Commit processor crashed " + errorCode);
```

Comment:
Could you please adopt Kfir's test case ? its in the patch he aploaded in the JIRA.